internal_workflow_client.go

```diff
@@ -63,6 +64,16 @@ type (
 	}
 )
 
+// RegisterWorkflow registers the workflow function with options
```

Comment:
minor: add comments about the parameters as well like rest of methods in wcclient. Mostly user lands on this class.